tests/ert/unit_tests/config/test_model_config.py

```diff
@@ -61,3 +64,13 @@ def test_that_invalid_time_map_file_raises_config_validation_error(tmpdir):
 
         with pytest.raises(ConfigValidationError, match="Could not read timemap file"):
             _ = ModelConfig.from_dict({ConfigKeys.TIME_MAP: "time_map.txt"})
+
+
+def test_warning_when_full_disk(tmp_path):
```

Comment:
Can this test be parameterized over (percentage, freespace, expectation) to also test the conditions where we should not trigger the warning?